test/libsolidity/semanticTests/functionCall/external_call.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity >= 0.6.0;
+//pragma experimental ABIEncoderV2;
+
+contract C {
+    function g(uint n) external pure returns (uint) {
+        return n + 1;
+    }
+
+    function f(uint n) public view returns (uint) {
+        return this.g(2 * n);
+    }
+
+    // h(uint256[],uint256): 0x40, 2, 3, 1, 2, 3 -> 0x20, 3, 2, 4, 6
```

Comment:
Can you remove this and add a test that uses statically-sized arrays?

Reply:
I added the test, but it fails due to `calldata not yet implemented!`

Reply:
```
    function h(uint[4] calldata n) external pure returns (uint) {
        return n[0] + n[1] + n[2] + n[3];
    }

    function i(uint[4] memory n) public view returns (uint) {
        return this.h(n) * 2;
    }
}

// ====
// compileViaYul: also
// ----
// g(uint256): 4 -> 5
// f(uint256): 2 -> 5
// h(uint256[4]): 1, 2, 3, 4 -> 10
// i(uint256[4]): 1, 2, 3, 4 -> 20
```